### DISCUSSION TITLE
Suggest: Add a way to manually disable some audio output in the seetings

### DISCUSSION THREAD

**Discussion by xx45464563** [File-New-Project/EarTrumpet#1030] [Ideas]
As written above
Sometimes I felt not very happy to see such a messy menu with so many useless thing 
Such as my monitor output, I simply don't use!
Is there any possibilities that can let me disable some output in the settings?
![image](https://user-images.githubusercontent.com/39402555/150056012-34ff1b70-fbfd-4cbe-ae15-9aaa70485537.png)
Here, the top three is what I don't need.

**Comment by xingyunjohn1** [comment]
Click the down arrow on the right top of your screenshot, then those will hide.

**Reply by xx45464563** [reply]
But I have two output at the same time 
for exp: I use my main speaker to play music   while I use my alternative output (headphone) to play game

**Reply by xx45464563** [reply]
就很离谱，我要让耳机和喇叭放两个不一样的流啊
然后比如那个nVidia broadcast，  我压根不用的，留在那里干哈？


It's outrageous that I have to have two different streams for the headphones and the speakers.
Then there's the nVidia broadcast, which I don't even use, so what's the point of leaving it there?

**Reply by xingyunjohn1** [reply]
Try to disable those unnecessary output devices in classic audio setting panel.

**Reply by Phoenix616** [reply]
Disabling outputs is not necessarily always what you want though. Some people (like me) might be using additional (virtual) audio devices to properly split up different programs for recording or to different outputs (with external usb sound cards) but they are (almost) never needed for adjusting volumes so they massively clutter that menu. E.g. [like this](https://i.moep.tv/gw39VX4Z.mp4).

**Comment by Phoenix616** [comment]
Duplicate of https://github.com/File-New-Project/EarTrumpet/discussions/895 btw.
